src/components/connected/login.js

```diff
@@ -1,9 +1,9 @@
 import React, { PropTypes, Component } from 'react'
 import { connect } from 'react-redux'
-import { login } from 'modules/auth'
+import { login } from '../../modules/auth'
 
 /* Components */
-import LoginForm from 'views/login'
```

Comment:
In 4f0c5c1, we introduced webpack's aliases to avoid the use of relative routes. If this is causing any problems, please file a separate issue so we can investigate why.